src/__swaps__/screens/Swap/hooks/useSwapWarning.ts

```diff
@@ -128,16 +128,7 @@ export const useSwapWarning = ({
       const nativeAmountImpact = subWorklet(Number(inputNativeValue).toString(), Number(outputNativeValue).toString());
```

Comment:
we can remove the `Number(x).toString() conversions` conversions now and just pass along the `inputNativeValue` etc to the `subWorklet` - we can also remove the `TODO` on line 127

Reply:
same thing for line 129